mktrayicon.c

```diff
@@ -372,30 +372,30 @@ gpointer watch_fifo(gpointer argv)
 				 */
 				int straight = 0;
 				int bars = 0;
-				int qEven = 1; // if single quotes are even -> we don't count anything 
+				int outQuoted = 1; // if single quotes are even -> we are outQuoted -> we can count 
```

Comment:
I think `outQuoted` is a really hard-to-read variable name. My vote would be for inverting the booleans and calling it `inQuote` instead.

Reply:
I will change that